pipeline/terraform/modules/stack/service_work_batcher.tf

```diff
@@ -59,7 +59,7 @@ module "batcher_lambda" {
       module.router_path_output_topic.arn,
       module.path_concatenator_output_topic.arn,
     ]
-    visibility_timeout_seconds = (local.wait_minutes + 5) * 60
+    visibility_timeout_seconds = (local.wait_minutes + 9) * 60
```

Comment:
Good catch. I think perhaps `wait_minutes` is actually not so useful in the context of the lambda, and this value should be the maximum runtime of the lambda? 

If we push the `lambda_timout` into locals, we can align both values, presuming the visibility timeout should always be at least as long the maximum execution time.

Also should we set `flush_interval_minutes` & `max_processed_paths` to zero for the lambda with the comment that they are not used in the lambda implementation?